src/Compilers/VisualBasic/Portable/BoundTree/BoundNodes.xml

```diff
@@ -247,6 +247,49 @@
         <Field Name="NamespaceSymbol" Type="NamespaceSymbol"/>
     </Node>
 
+    <!-- Represents Typeof(PrivateImplementationDetails), which is a special case because PrivateImplementationDetails has no language-level symbol.
+         Used in the synthesized static constructor for PrivateImplementationDetails. -->
+    <Node Name="BoundTypeOfPrivateImplementationDetails" Base="BoundTypeOf">
+    </Node>
+
+    <!-- Represents the raw metadata token index value for a method definition.
+         Used by dynamic instrumentation to index into tables or arrays of per-method information. -->
+    <Node Name="BoundMethodDefIndex" Base="BoundExpression">
+        <!-- Non-null type is required for this node kind -->
+        <Field Name="Type" Type="TypeSymbol" Override="true" Null="disallow"/>
+        <Field Name="Method" Type="MethodSymbol"/>
+    </Node>
+
+    <!-- Represents the maximum raw metadata token index value for any method definition in the current module. -->
+    <Node Name="BoundMaximumMethodDefIndex" Base="BoundExpression">
+        <!-- Non-null type is required for this node kind -->
+        <Field Name="Type" Type="TypeSymbol" Override="true" Null="disallow"/>
+    </Node>
+
+    <!-- Represents the dynamic analysis instrumentation payload array for the analysis kind in the current module.
+         Implemented as a reference to a field of PrivateImplementationDetails, which has no language-level symbol. -->
+    <Node Name="BoundInstrumentationPayloadRoot" Base="BoundExpression">
+        <Field Name="AnalysisKind" Type="Integer"/>
+        <!-- Non-null type is required for this node kind -->
+        <Field Name="Type" Type="TypeSymbol" Override="true" Null="disallow"/>
+        <Field Name="IsLValue" PropertyOverrides="true" Type="Boolean"/>
+    </Node>
+
+    <!-- Represents the GUID that is the current module's MVID.
+         Implemented as a reference to PrivateImplementationDetails.MVID, which has no language-level symbol. -->
+    <Node Name="BoundModuleVersionId" Base="BoundExpression">
+        <!-- Non-null type is required for this node kind -->
+        <Field Name="Type" Type="TypeSymbol" Override="true" Null="disallow"/>
+        <Field Name="IsLValue" PropertyOverrides="true" Type="Boolean"/>
+    </Node>
+
+    <!-- Represents a string encoding of the GUID that is the current module's MVID.
+         Implemented as a reference to a string constant that is backpatched after the MVID is computed. -->
+    <Node Name="BoundModuleVersionIdString" Base="BoundExpression">
```

Comment:
Yes, and I found an override that is missing in the C# implementation as well. Will add.